sdk/helper/random/rules.go

```diff
@@ -0,0 +1,82 @@
+package random
+
+import (
+	"fmt"
+
+	"github.com/mitchellh/mapstructure"
+)
+
+// Charset requires a certain number of characters from the specified charset.
+type Charset struct {
```

Comment:
Nit: consider adding `Rule` to the struct name

Reply:
I'm not opposed to this, though it's worth noting that I was intentionally making the struct name match the rule name.